src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+
+  &__image {
+    width: 160px;
```

Comment:
The width of the image should preferably not be fixed to ensure that the image scales well on different screen sizes. Consider using percentages or a more responsive unit like 'vw' for width.